.travis.yml

```diff
@@ -7,44 +7,9 @@ os: linux
 dist: focal
 
 jobs:
-  fast_finish: true
   include:
-    - name: "Test Rust"
-      language: rust
-      rust: stable
-      env:
-        - SCCACHE_VERSION=0.2.13
```

Comment:
How does caching work on github actions?

Reply:
caching looked a little more complicated so i didn't add it to this pr. will measure if sccache speeds up build and look more into caching in another pr.